parastell/parastell.py

```diff
@@ -457,6 +412,63 @@ def export_dagmc(
                 **kwargs
             )
 
+    def build_full_model(self, invessel_build, magnet_coils, source_mesh):
+        """Use all of the input to make a complete stellarator model
+        with invessel components, magnets, and a source mesh
+        """
+
+        # In-Vessel Build
+        self.construct_invessel_build(
+            invessel_build['toroidal_angles'],
+            invessel_build['poloidal_angles'],
+            invessel_build['wall_s'],
+            invessel_build['radial_build'],
+            **invessel_build
+        )
+
+        # Magnet Coils
+        self.construct_magnets(
+            magnet_coils['coils_file'],
+            magnet_coils['cross_section'],
+            magnet_coils['toroidal_extent'],
+            **magnet_coils
+        )
+
+        # Source Mesh
+        self.construct_source_mesh(
+            source_mesh['mesh_size'],
+            source_mesh['toroidal_extent'],
+            **source_mesh
+        )
+
+    def export_full_model(self, export_dir, invessel_build, 
+                          magnet_coils, source_mesh, dagmc_export):
+    
+        export_cad_to_dagmc = invessel_build.get('export_cad_to_dagmc', False)
+        dagmc_filename = invessel_build.get('dagmc_filename', 'dagmc')
+
+        self.export_invessel_build(
+            export_dir=export_dir,
+            export_cad_to_dagmc=export_cad_to_dagmc,
+            dagmc_filename=dagmc_filename
+        )
+        
+        self.export_magnets(
+            export_dir=export_dir,
+            **(filter_kwargs(magnet_coils,mc.export_allowed_kwargs))
+        )
```

Comment:
Why do we handle `export_invessel_build` and `export_magnets` differently? They handle keyword arguments very similarly.